### PR TITLE
chore(github): Add Bug Report issue template

### PR DESCRIPTION
**Pull request by CITguy**
## LGTM's
- [x] Dev LGTM
- [x] Zoom LGTM

Once merged, we'll be able to create a new bug report using a link in the documentation's global header.

screen shot 2018-06-22 at 4 32 39 pm

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+---
+
+## Describe the Bug
+A clear and concise description of what the bug is.
+
+## Steps to Reproduce
+Steps to reproduce the behavior:
+
+1. Go to '...'
+2. Click on '....'
```
Comment (nimbinatus) [inline .github/ISSUE_TEMPLATE/bug-report.md]: 3 dots for ellipsis.

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+3. Scroll down to '....'
```
Comment (nimbinatus) [inline .github/ISSUE_TEMPLATE/bug-report.md]: ditto

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+4. See error
```
Comment (nimbinatus) [inline .github/ISSUE_TEMPLATE/bug-report.md]: Add ellipsis. And profit.

**Comment by CITguy** [comment]
FYI: Here's what it looks like when the user clicks "Report Bug"

screen shot 2018-06-22 at 5 15 19 pm

**Pull request merged**